apertium-tur-uzb.tur-uzb.lrx

```diff
@@ -1,18 +1,112 @@
 <rules>
 
-<rule weight="1"> 
-    <match lemma="boyunca" tags="post">
-    <select lemma="bo'ylab" tags="post"/>
-    </match>
-</rule>
-
-<rule weight="2"> 
-    <match lemma="*" tags="num"/>
-    <match lemma="*" tags="n.*"/>
-    <match lemma="boyunca" tags="post*">
+  <rule weight ="1">  <!-- default -->
+  <match lemma="güzel" tags="adj">
+    <select lemma="goʻzal" tags="adj"/>
+  </match>
+  </rule>
+  <rule weight ="2">
+  <match lemma="çok"/> <!-- if güzel comes after çok, it means "very good"(mostly) (Ex: Hava bugün çok güzel) -->
+  <match lemma="güzel" tags="adj">
+    <select lemma="ajoyib" tags="adj"/>
+  </match>
+  </rule>
+
+
+  <rule weight ="1">  <!-- default -->
+  <match lemma="boyunca" tags="post">
+    <select lemma="boʻylab" tags="post"/>
+  </match>
+  </rule>  
+  <rule weight ="2"> <!-- if boyunca comes after a number followed by a noun (Ex: iki gün boyunca) --> <!-- Currently not working -->
```

Comment:
Ah, is still not working? but worked for me.

Reply:
It's working for me too.